src/content_mapblock.cpp

```diff
@@ -1291,8 +1270,7 @@ void MapblockMeshGenerator::drawNode()
 	else
 		light = getInteriorLight(n, 1, nodedef);
 	switch (f->drawtype) {
-		case NDT_LIQUID:            drawLiquidNode(false); break;
-		case NDT_FLOWINGLIQUID:     drawLiquidNode(true); break;
+		case NDT_FLOWINGLIQUID:     drawLiquidNode(); break;
```

Comment:
Ah now I see why this can be removed. That's quite new land to me, thus thanks for pointing this out :)